sdk/azurestack/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <IsLegacyMgmtProject Condition="'$(IsLegacyMgmtProject)' == ''">true</IsLegacyMgmtProject>
```

Comment:
I wonder if we want to insert this in all the management csproj files.  I am wondering if we are disabling any build scenarios, like building from the local directory for management projects.

Reply:
You could manually put them in all project files but I think that is overkill. The way we have it setup now this should work for building these projects in any context. Even if you build from inside the project directory this should still get imported correctly.